rpc/src/lib.rs

```diff
@@ -29,7 +29,7 @@ enum ZeroTrace {
 /// - Just the hash.
 ///
 /// We only need the latter.
-const BLOCK_WITHOUT_FULL_TRANSACTIONS: bool = false;
+const BLOCK_WITHOUT_FULL_TRANSACTIONS: BlockTransactionsKind = BlockTransactionsKind::Hashes;
```

Comment:
I'd rename the constant at this point, it's not really consistent with the type anymore.

Reply:
Would someting like `BLOCK_TRANSACTIONS_HASHES_ONLY` work?

Reply:
Does it make sense to have a constant anymore? I think using the enum itself reads easier.